src/hera/volumes.py

```diff
@@ -115,6 +116,46 @@ def get_mount(self) -> V1VolumeMount:
         return V1VolumeMount(name=self.name, mount_path=self.mount_path)
 
 
+class SecretVolume(BaseModel):
+    """A volume representing a secret. This can be used to mount secrets to paths inside a task
+
+    Attributes
+    ----------
+    name: str
+        The name of the volume
+    secret_name: str
+        The name of the secret existing in the task namespace
+    mount_path: str
+        The mounting point in the task e.g /mnt/my_path. The secrets will be mounted to this path, with the names
+        being used as file names, and the values in those files
+    """
+
+    name: str
+    secret_name: str
+    mount_path: str
```

Comment:
This looks awesome! 

What do you think about taking the opportunity to help users by making `mount_path` `Optional`? It might be very nice to have the `mount_path` default to `f"tmp/secrets/{self.secret_name}"` when the `mount_path` is `None`. Very open to pushback on this idea.

Reply:
That sounds very reasonable. Thank you for explaining the use case 🙂